src/main/java/com/mrdb/controller/MainController.java

```diff
@@ -0,0 +1,46 @@
+package com.mrdb.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.util.JSONPObject;
+import com.mrdb.entities.UserEntity;
+import com.mrdb.entities.UserEntityRepository;
+import org.apache.tomcat.util.json.JSONParser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+
+@RestController
+@RequestMapping("/api/users")
+public class MainController {
+    @Autowired private UserEntityRepository repo;
+
+    @PostMapping("/login")
+    public String login(@RequestBody UserEntity userDetails) {
+        String res = "Ok";
+        String userName = userDetails.getUserName();
+        String password = userDetails.getPassword();
+        UserEntity user = repo.findByUserName(userName);
+        if(null != user && password.equalsIgnoreCase(user.getPassword())) {
```

Comment:
`equalsIgnoreCase` Do you understand the implications of ignoring the case here?

Reply:
Sir yes sir,
I just want to check weather it was working or not, I thought of removing it, but forgot to. Instead I am just querying directly from the DB with "findByUserNameAndPassword". In the future obviously we want to use an encryption to save password to db.

Reply:
Well, I am not talking about encryption though. Ignoring case : a password of `hello` will be same as `Hello`